includes/class-woocommerce.php

```diff
@@ -812,6 +830,8 @@ public function deactivated_plugin( $filename ) {
 	 * Get queue instance.
 	 *
 	 * @return WC_Queue_Interface
+	 *
+	 * @deprecated 4.3.0 Use the container to get an instance of WC_Queue_Interface instead.
```

Comment:
This will need updated to the version we merge this in, likely 4.4.0?

Reply:
Sure, I just added the next version number in sight with the idea of adjusting it before actually merging the PR.